templates/_includes/analytics.html

```diff
@@ -33,3 +33,9 @@
         })();
     </script>
 {% endif %}
+
+{% if GOAT_COUNTER %}
+<script data-goatcounter="https://{{ GOAT_COUNTER }}.goatcounter.com/count"
```

Comment:
People can use custom domains (e.g. stats.example.com) and self-host GoatCounter. Personally I'd recommend that people fill in their entire URL in this variable since that'll cover all use cases:

    <script data-goatcounter="{{ GOAT_COUNTER }}" async src="//gc.zgo.at/count.js">

I can perhaps add an easier "external integration" copy/paste in the backend for this; as the WordPress plugin also works like this.

Reply:
Wow! So glad and impressed to see you review the code for us! ❤️ We will make the recommended changes.

BTW, I love GoatCounter and have been advocating it in my circle. More power to you!